src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -290,14 +290,14 @@ interface SdJwtVerifier<JWT> {
      * @return the verified SD-JWT, if valid. Otherwise, method could raise a [SdJwtVerificationException]
      * The verified SD-JWT will contain a [JWT][SdJwt.jwt] as both string and decoded payload
      */
-    suspend fun verifyIssuance(
+    suspend fun verify(
         jwtSignatureVerifier: JwtSignatureVerifier<JWT>,
         unverifiedSdJwt: String,
     ): Result<SdJwt<JWT>>
 
     /**
-     * Verifies an SD-JWT in JWS JSON general of flattened format as defined by RFC7515 and extended by SD-JWT
-     * specification
+     * Verifies an SD-JWT serialized using JWS JSON serialization (either general or flattened format) as defined by RFC7515
```

Comment:
Add also

Typically, this is useful to Holder that wants to verify an issued SD-JWT or to Verifier
that wants to verify a presented SD-JWT in case the KB-JWT [must not be present]KeyBindingVerifier.MustNotBePresent]